### PR TITLE
feat: code gerenciamento de contas

### PR DESCRIPTION
**Pull request by artur-seppa**

### CODE REVIEW

File: src/model/Account.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    if (initialBalance < 0) {
+      throw new Error('O saldo inicial deve ser um número não negativo.');
+    }
+
+    if (isNaN(initialBalance) || typeof initialBalance === 'string') {
+      throw new Error('O valor de saldo deve ser apenas de tipo Number.');
+    }
+
```
Comment (fernandollisboa) [inline src/model/Account.js:21]: this works for creating an account directly, but in the terminal, when the name and initialBalance are empty, it results in unexpected behavior:
image

File: src/collection/AccountCollection.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+export class AccountCollection {
+    constructor() {
+        this.accounts = [];
+    }
```
Comment (fernandollisboa) [inline src/collection/AccountCollection.js]: i believe `this.account` should be private and only be accessible through methods

File: tests/account.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+// account.test.js
```
Comment (fernandollisboa) [inline tests/account.test.js:1]: 👁️ 👄 👁️

File: tests/account.test.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+// account.test.js
+import { describe, it, expect } from 'vitest';
+import { Account } from '../src/model/Account.js';
+
+describe('Account', () => {
+  it('Criacao de uma conta valida', () => {
```
Comment (fernandollisboa) [inline tests/account.test.js]: sadly, our language is not commonly used for test documentation, so I strongly suggest using english 🇬🇧 for test descriptions

File: tests/account.test.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    const account = new Account('Artur Seppa');
+    
+    expect(account.getBalance()).toBe(0);
+  });
+
+  it('Erro para name vazio', () => {
+    expect(() => new Account('')).toThrow('O nome não pode ser vazio.');
+    expect(() => new Account('  ')).toThrow('O nome não pode ser vazio.');
+    expect(() => new Account()).toThrow('O nome não pode ser vazio.');
+  });
```
Comment (fernandollisboa) [inline tests/account.test.js]: nice testes! but there are still some edge cases that could be tested, such as attempting to update an account with an empty name

**Review by fernandollisboa — COMMENTED** [review]
nice! i've left some comments
